lib/platform.js

```diff
@@ -32,6 +42,7 @@ MisfitBoltPlatform.prototype.accessories = function(callback) {
     accessory.initialize(function() {
       if (totalAccessories == configs.length) {
```

Comment:
we should probably also check for `done !== true` to prevent running the callback twice.